usrp_drivers/n200/n200_driver.cpp

```diff
@@ -24,271 +24,339 @@
 #include "utils/driver_options/driveroptions.hpp"
 #include "usrp_drivers/n200/usrp.hpp"
 #include "utils/protobuf/driverpacket.pb.h"
-#include "utils/protobuf/computationpacket.pb.h"
+#include "utils/protobuf/rxsamplesmetadata.pb.h"
 #include "utils/shared_memory/shared_memory.hpp"
 
-
-std::vector<size_t> make_channels(const driverpacket::DriverPacket &dp) {
-  std::vector<size_t> channels(dp.channels_size());
-  for (int i = 0; i < dp.channels_size(); i++) {
-    channels[i] = i;
+#ifdef DEBUG
```

Comment:
For the timing code in the while loop  below, (like time_diff = std::chrono::duration_cast .blah blah) should they be wrapped in an #ifdef DEBUG ? since they are strictly for profiling?

Reply:
made a new macro for profiling